modin/engines/dask/task_wrapper.py

```diff
@@ -26,5 +47,18 @@ def deploy(cls, func, num_returns, kwargs):
 
     @classmethod
     def materialize(cls, future):
+        """
+        Materialize data matching ``future`` object.
```

Comment:
```suggestion
        Materialize data matching `future` object.
```
numpydoc says reference parameters with single backicks